config/routes.rb

```diff
@@ -1,6 +1,8 @@
 Rails.application.routes.draw do
-  devise_for :users
   root 'home#index'
+  put 'users/edit'
```

Comment:
šo vari izņemt un izmantot standarta rails routes fičas
Tev jāpievieno resources :users, only: [:edit, :update, :destroy]

Un tad talak vajadzes nomainit linku users_edit_path uz edit_user_path(current_user)
Tas ir rails conventional. Vairak par šo magiku:
https://guides.rubyonrails.org/getting_started.html#resourceful-routing

Reply:
Es ievietoju `resources :users , only: [:edit , :update]` , bez destroy , jo tas konflikteja ar devise sign_out , un vins uztvera sign_out ka id , tapec es atstaju veco route , tad strada gan account delete , gan sign_out

Reply:
@ErnestsEida vajadzētu pamēģināt nokonfigurēt lai devise izmanot citu namespace / default kontrolierus priekš šīm metodēm.

By default devise parasti izmantoja sessions namespace, kaut kas nešķiet pareizi, ja tagad izmanto Users#destroy pie sign out....

Reply:
https://github.com/ErnestsEida/R-edited/pull/34